src/components/LineLike/LineLike.tsx

```diff
@@ -8,34 +8,33 @@ import './LineLike.scss'
 export const LineLike = () => {
     
     const [likeValue, setLikeValue] = useState(0)
-    const [likeBoolValue, setLikeBoolValue] = useState(false)
 
     const [dislikeValue, setDislikeValue] = useState(0)
-    const [dislikeBollValue, setDislikeBoolValue] = useState(false)
-    
+ 
     const [bookValue, setBookValue] = useState(false)
 
-    function addLikeOrDislike(set1: any, set2: any, value1: number, value2: boolean): void{
-        if(!value2){
-            set1(value1 + 1);
-            set2(true);
-        }else{
-            set1(value1 - 1);
-            set2(false);
+    function addLikeOrDislike(set: any, value: any): void{
+        setLikeValue(0)
+        setDislikeValue(0)
+        if(value !== 0) {
+            set(0)
+        }else {
+            set(1)
         }
+        
```

Comment:
Так как количество лайков и дизлайков не должно быть ограничено одним. Лучше реализовать функцию следующим образом
```suggestion
    const addLike = () => {
      setLikeValue((previousValue) => previousValue + 1)
    }
     const addDislike = () => {
       setDislikeValue((previousValue) => previousValue + 1)
    }
```
а в кнопки уже установить эти функции
`<button onClick={addLike} …/>`
`<button onClick={addDislike} …/>`